src/index.html

```diff
@@ -2,12 +2,55 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <article class="product" data-qa="card">
+      <img
+        class="product__image"
+        src="./images/imac.jpeg"
+        alt="APPLE A1419 iMac 27 inches Retina 5K Monoblock (MNED2UA/A)"
+      >
+      <div class="product__information">
+        <h2 class="product__tittle">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+        <div class="product__code">
+          Product code: 195434
+        </div>
+        <div class="product__reviews">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          Reviews: 5
+        </div>
+        <div class="product__price">
+          <div class="product__text">Price:</div>
+          <div class="product__cost">$2,199</div>
+        </div>
+      </div>
+      <a
+        href="#"
+        class="link"
```

Comment:
```suggestion
        class="product__link"
```
It should be an element of product block